lib/eslintTypescriptConfig.json

```
@@ -1,7 +1,7 @@
 {
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
-    "project": ["./tsconfig.json"]
+    "project": true
```

The doc you linked mentions:

> note
We now recommend using [projectService](https://typescript-eslint.io/packages/parser/#projectservice) instead of project for easier configuration and faster linting.

So maybe we can go for 
```suggestion
    "projectService": true
```

---

Ah yea that's for version 8. We're stuck on version 7 for now, so we have to use `project` which is the best option available for that version.

---

Ah missed that sorry! Looking good tehn!

---

No problem, I have a plan to get us on 8, but it's blocked on the eslint AirBNB packages updating their dependencies. It will coincide with ESLint v9 support too which is handy. I'm tracking it here: https://github.com/airbnb/javascript/issues/2961